frigate/config.py

```diff
@@ -210,6 +210,14 @@ class FilterConfig(FrigateBaseModel):
     max_area: int = Field(
         default=24000000, title="Maximum area of bounding box for object to be counted."
     )
+    min_ratio: float = Field(
+        default=0,
+        title="Minimum ratio of bounding box's width/height for object to be counted.",
+    )
+    max_ratio: float = Field(
+        default=24000000,  # @todo: Is there a way to make this `math.inf` but still load from YAML?
```

Comment:
I think this is a fine default. Objects can't be less than 1px tall and nobody is going to have a camera that is 24000000  pixels wide.